sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -67,3 +69,10 @@ export class CreateInstitutionDto {
   @IsNotEmpty()
   postalCode: string;
 }
+
+export class InstituteDto extends PartialType(CreateInstitutionDto) {}
```

Comment:
Please consider renaming it to "Institution" instead of "Institute".

Reply:
DONE